setup.cfg

```diff
@@ -25,6 +25,8 @@ python_requires = >=3.6
 install_requires =
     traitlets
     entrypoints
+    importlib_resources
```

Comment:
You're not using this yet.

Reply:
Sorry, had just woken up, can only do so much before the first pot of coffee.